wpimath/generate_quickbuf.py

```diff
@@ -22,7 +22,8 @@ def generate_quickbuf(
                 f"--quickbuf_out=gen_descriptors=true:{output_directory.absolute()}",
                 f"-I{absolute_filename.parent}",
                 absolute_filename,
-            ]
+            ],
+            check=True,
```

Comment:
Nit: I usually do `subprocess.check_call` instead of `subprocess.run`

Reply:
Done in https://github.com/wpilibsuite/allwpilib/pull/8069